Zhana_Konsulova_7MI0800248_group6/Prakt2_zad2/Prakt2_zad2/Prakt2_zad2.cpp

```diff
@@ -0,0 +1,90 @@
+// Prakt2_zad2.cpp : This file contains the 'main' function. Program execution begins and ends there.
+//
+
+#include <iostream>
+
+
+struct Pair
+{
+    int first, second;
+    void Read()
+    {
+        std::cin >> first >> second;
+    }
+    void Print() const
+    {
+        std::cout << "(" << first << ", " << second << ")" << std::endl;
+    }
+};
+
+bool IsFirstPairBigger(const Pair& p1,const Pair& p2)
+{
+    return
+        p1.first > p2.first ||
+        (p1.first == p2.first && p1.second > p2.second);
+}
+
+Pair TheBiggest(Pair arr[100])
+{
+    Pair theBiggest = arr[0];
+    for (int i = 1; i < 100; i++)
+    {
+        if (IsFirstPairBigger(arr[i], theBiggest))
+            theBiggest = arr[i];
+    }
+    return theBiggest;
+}
+
+
+
+void Swap(Pair& p1, Pair& p2)
```

Comment:
fyi but there's the [std::swap](https://en.cppreference.com/w/cpp/algorithm/swap) too :P